drivers/serial/uart_ite_it8xxx2.c

```diff
@@ -0,0 +1,959 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Created on: 2015/06/17
+ *     Author: Jyunlin Chen
+ */
+
+#include <errno.h>
+#include <kernel.h>
+#include <arch/cpu.h>
+#include <zephyr/types.h>
+#include <soc.h>
+
+#include <init.h>
+#include <toolchain.h>
+#include <linker/sections.h>
+#include <drivers/uart.h>
+#include <sys/sys_io.h>
+
+/**
+ * struct declaration
+ */
+#define t_uart_0                0
+#define t_uart_1                1
+#define t_uart_max_idx          2
+
+enum _T_UART_BAUDRATE_INDEX_ {
+	t_br_4800   = 0,
```

Comment:
should we drop the =0? Is this enum ever serialized, if not,  let's just let the compile assign values

Reply:
It does get used as indices to an array (mBaudrateTb[]) later on, so we should probably keep the =0

Reply:
Hmm, I still don't think that it worth keeping the =0, but I won't get stuck on this.